givitsite/friendfeed/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.requestItem, name = 'requestItem'),
+   
+]
```

Comment:
"no newline at end of file" here too. :sob:

Reply:
fixed